hack/gen-flavors.sh

```diff
@@ -17,10 +17,17 @@
 set -o errexit
 set -o nounset
 set -o pipefail
-#set -o verbose
+# set -o verbose
 
 root=$(dirname "${BASH_SOURCE[0]}")/..
+script_invocation_dir="$(pwd)"
 kustomize="${root}/hack/tools/bin/kustomize"
+
+# go to root and install kustomize if missing
+cd ${root} && make "${kustomize##*/}"
+
+# come back to script's invocatio dir
```

Comment:
```suggestion
# come back to script's invocation dir
```

Reply:
Updated via commit f6e9cdbbcd09f7a809952d084d24006b31c2438e but that commit umbrellas to two files, sorry about that.